### PR TITLE
First very basic bias correction functions

### PR DESCRIPTION
**Pull request by huard**
* **Please check if the PR fulfills these requirements**
- [x] This PR addresses an already opened issue (for bug fixes / features)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] HISTORY.rst has been updated
- [ ] bumpversion (minor / major / patch) has been called
- [ ] Tags have been pushed

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, etc.)
Basic quantile mapping bias correction functions. 

* **Does this PR introduce a breaking change?** (Has there been an API change?)
No

* **Other information**:

* When merging, please put `fixes #{issue number}` in your comment to auto-close the issue that your PR addresses. Thanks!

### CODE REVIEW

**Comment by huard** [comment]
Ideally Pat would also review so we start in the right direction.

File: tests/test_indices.py [hunk 1]
```diff
@@ -463,6 +473,16 @@ def test_run_start_at_0(self, pr_series):
         assert out[0] == 10
 
 
+class TestMaximumConsecutiveSummerDays:
+    def test_simple(self, tasmax_series):
+        a = np.zeros(365) + 273.15
+        a[5:15] += 30
+        tx = tasmax_series(a, start="1/1/2010")
+        out = xci.maximum_consecutive_summer_days(tx, thresh="25 C", freq="M")
```
Comment (tlogan2000) [inline tests/test_indices.py]: Didn't we change the name of this recently?

File: xclim/indices/_threshold.py [hunk 2]
```diff
@@ -649,6 +650,46 @@ def maximum_consecutive_dry_days(pr, thresh="1 mm/day", freq="YS"):
     return group.apply(rl.longest_run, dim="time")
 
 
+@declare_units("days", tasmax="[temperature]", thresh="[temperature]")
+def maximum_consecutive_summer_days(tasmax, thresh="25 degC", freq="YS"):
```
Comment (tlogan2000) [inline xclim/indices/_threshold.py]: Name change?

File: tests/test_indices.py (hunk 1 above)
Comment (huard) [reply tests/test_indices.py]: No, it's just that I did the initial checkout on branch fix-61, not master. Once we merge fix-61, that should go away.

File: xclim/indices/_threshold.py (hunk 2 above)
Comment (huard) [reply xclim/indices/_threshold.py]: Idem. no change to this in this branch. sorry about the confusion.

**Comment by huard** [comment]
I've no problem if the bias correction code stays in a branch for a while. Let's get comments from Pat. There is no rush in merging this.

**Comment by Balinus** [comment]
I'd say that if all is missing is detrending and moving window, it should be merged and tagged as experimental in the docs and function help (?). This way, users would at least test the core functionality of q-q mapping.

**Comment by huard** [comment]
Interpolation over the monthly correction factor is already in there (interp=True).

**Comment by huard** [comment]
Review expected by Sept. 17.

**Comment by huard** [comment]
Moved to xsd.

**Pull request closed without merging**